code/datums/supply_packs.dm

```diff
@@ -410,8 +410,7 @@
 					/obj/item/device/reagentscanner)
 	cost = 500
 	containertype = /obj/storage/secure/crate/plasma
```

Comment:
I believe this crate is still set to be a secure one. So the contents will be locked and no one will be able to open it. Could be wrong though.

Reply:
You still have to "Unlock" it, but anybody can unlock it now. Only did it this way cause of lack of knowledge, and I figured using a standard metal crate doesn't fit the theme.